tools/devctr/pyproject.toml

```diff
@@ -5,22 +5,28 @@ description = "Firecracker python dependencies"
 authors = ["Your Name <you@example.com>"]
 
 [tool.poetry.dependencies]
+aws-embedded-metrics = "*"
 black = "*"
 boto3 = "*"
 botocore = "*"
 dataclasses = "*"
 gitlint = "0.16.0"
+less = "*"
 nsenter = "*"
+openapi-spec-validator = "*"
 pandas = "*"
 psutil = "5.7.0"
 pylint = "*"
 pytest = "*"
+pytest-json-report = "*"
 pytest-timeout = "*"
+pytest-xdist = "*"
 python = "3.10.4"
 PyYAML = "*"
 requests = "*"
 requests-unixsocket = "*"
 retry = "*"
+setproctitle = "*"
 typing-extensions = "*"
 # We need to fix the gitlint version since version 0.17.0 has a problem:
 # https://pythonissues.com/issues/1410824
```

Comment:
I know it's not introduced in your change, but could you move the comment to the gitlint line?

Reply:
yes, sure, I'll move it before the gitlint line for better visibility